cmd/root.go

```diff
@@ -21,18 +22,21 @@ var (
 	cfgFile   string
 	projectID string
 	opts      = sdk.Options{}
+	verbose   *bool
 )
 
 // NewRootCmd creates a new root command
 func NewRootCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
-		Use: "miactl",
+		Use:          "miactl",
+		SilenceUsage: true,
```

Comment:
I just find annoyingly that every time a command return an error the usage is shown, even though the command was perfectly fine (e.g. wrong password in login, but command syntax was fine).

Since this is my personal opinion, it can be removed and revert the cli to its default behaviour.